ios/ReactConf2017/MobileCenter-Config.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>AppSecret</key>
+    <string>3a8e7774-5b7d-4f2b-b49e-434116446e5e</string>
```

Comment:
Fair enough, maybe calling it something like AppId might be more clear? Thanks for clarifying. This obviously isn't a blocker for merging.